dotnet4/map.jinja

```diff
@@ -1,6 +1,7 @@
 # Initialize 'dotnet4' dictionary with default settings for the dotnet4 formula
 {% set dotnet4 = {
-    'version'     : '4.6.01590',
+    'version': '4.6.01590',
```

Comment:
Does version need to be most recent, 4.7.02558 ?

Reply:
No, everything works fine with the earlier version. If the user wants a newer version, they set the desired version via pillar.